web/js/containers/sidebar/layers-container.js

```diff
@@ -0,0 +1,201 @@
+            ref={provided.innerRef}
+            {...provided.draggableProps}
+            {...provided.dragHandleProps}
+          >
+            <LayerList
+              title={groupName}
+              groupId={groupName}
+              collapsed={collapsed}
+              toggleCollapse={toggleCollapse}
+              compareState={compareState}
+              layerSplit={overlays.length}
+              layers={layersForGroup}
+            />
+          </li>
+        )}
+      </Draggable>
+    ));
+  };
+
+  const renderOverlayGroups = () => (
+    <DragDropContext onDragEnd={onDragEnd}>
+      <Droppable
+        droppableId="layerGroup"
+        type="overlayGroups"
+        direction="vertical"
+      >
+        {(provided, snapshot) => (
+          <ul
+            ref={provided.innerRef}
+            {...provided.droppableProps}
+          >
+            {overlayGroups.map(renderLayerList)}
+            {provided.placeholder}
+          </ul>
+        )}
+      </Droppable>
+    </DragDropContext>
+  );
+
+  const scrollContainerStyles = {
+    maxHeight: `${height}px`,
+    overflowY: 'auto',
+    paddingBottom: '4px',
+    minHeight: '100px',
+  };
+
+  return isActive && (
+    <>
+      <div id="layers-scroll-container" style={scrollContainerStyles}>
+        <div className="layer-container sidebar-panel">
+
+          {groupOverlays ? renderOverlayGroups() : (
+            <LayerList
+              title="Overlays"
+              groupId="overlays"
+              compareState={compareState}
+              collapsed={overlaysCollapsed}
+              toggleCollapse={() => toggleOverlaysCollapsed(!overlaysCollapsed)}
+              layers={overlays}
+              layerSplit={overlays.length}
+            />
+          )}
+
+          <div className="layer-group-baselayers">
+            <LayerList
+              title="Base Layers"
+              groupId="baselayers"
+              collapsed={baselayersCollapsed}
+              toggleCollapse={() => toggleBaselayersCollapsed(!baselayersCollapsed)}
+              compareState={compareState}
+              layers={baselayers}
+              layerSplit={overlays.length}
+            />
+          </div>
+        </div>
+      </div>
+      <div className="group-overlays-checkbox">
+        <Checkbox
+          id="group-overlays-checkbox"
+          checked={groupOverlays}
+          onCheck={toggleOverlayGroups}
+          label="Group Similar Layers"
+        />
+      </div>
+    </>
+  );
+}
+
+const mapStateToProps = (state, ownProps) => {
+  const { compareState } = ownProps;
+  const { layers } = state;
+  const { groupOverlays } = layers[compareState];
+  const { baselayers, overlays } = getAllActiveOverlaysBaselayers(state);
+  const overlayGroups = groupOverlays ? getActiveOverlayGroups(state) : [];
+
+  return {
+    baselayers,
+    overlays,
+    overlayGroups,
+    groupOverlays,
+    activeLayersMap: getActiveLayersMap(state),
+    layerSplit: overlays.length,
```

Comment:
`layerSplit: overlays.length,`
This doesn’t appear to be used and/or logic is being repeated elsewhere (which may be necessary)

Reply:
Yup, I think I meant to just pass it to children as `overlays.length` to reduce props in this file then forgot to remove it.  Nice catch!